staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -127,6 +127,10 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.StringVar(&s.EncryptionProviderConfigFilepath, "experimental-encryption-provider-config", s.EncryptionProviderConfigFilepath,
 		"The file containing configuration for encryption providers to be used for storing secrets in etcd")
+
+	fs.BoolVar(&s.StorageConfig.DisableCompaction, "etcd-disable-compaction", s.StorageConfig.DisableCompaction,
```

Comment:
probably we should make it clear that if this option is enabled, user MUST compact the underlying etcd3 storage themselves.

Reply:
It makes sense. I'll update the description later. Thanks.